website/docs/how-to/how-to-download-login-history.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: How to download login history
+---
+
+:::info availability
+
+Login history is an enterprise feature available from Unleash 4.22 onwards.
+
+:::
+
+[Login history](../reference/login-history.md) enables Unleash admins to audit login events and their respective information, including whether they were successful or not.
+
+## Step 1: Navigate to the login history page {#step-1}
+
+Navigate to the _login history_ page in the admin UI (available at the URL `/admin/logins`). Use the _settings_ button in the navigation menu and select "login history".
+
+![The admin UI navigation settings submenu with the login history item highlighted.](/img/login-history-1.png)
+
+## Step 2: Click the "Download login history" button {#step-2}
+
+Use the "download login history" button to proceed with the download of the login history as CSV.
+
+![The login history table with the "download login history" button highlighted.](/img/login-history-2.png)
+
+## Deleting login history events
+
+After downloading the existing login history, you may want to delete the events from the database. This can be done by clicking the "Clear login history" button.
+
+You may also optionally delete single events by clicking the "delete" button in the respective row.
+
+![The login history table with the "clear login history" button highlighted.](/img/login-history-3.png)
```

Comment:
That's a valid concern. I'm not sure if "deleting your login history" warrants a separate guide, and it felt somehow connected in the flow: After downloading your login history, maybe you don't need it in Unleash anymore.

Reply:
Right. I'd argue that it does. If it's important enough that we want to dedicate a section to it, then it probably warrants its own guide. But there's two separate items here, right:
- how to clear the history
- how to remove individual events.

That said, I don't want this to be a blocker for merging this pr, but maybe something that can be cleaned up later?

Reply:
I'm not sure if it's that important, since the retention logic will automatically delete old login events anyways.
How do you feel about removing that section for now? Would that be clearer?

Reply:
Yeah, I think that's a good suggestion. The fact that you can delete them is already mentioned in the reference docs, so that should be good. If we get questions about how to do it, then we can write this guide!

Reply:
Thanks, addressed in https://github.com/Unleash/unleash/pull/3261/commits/670c66b5419f4eb6039b3102c9c7304130b71abc